src/server/pkg/testpachd/real_env.go

```diff
@@ -40,6 +41,18 @@ const (
 // environment in order to spin up pipelines, which is not yet supported by this
 // package, but the other API servers work.
 func WithRealEnv(cb func(*RealEnv) error, customConfig ...*serviceenv.PachdFullConfiguration) error {
+	// TODO: This means StorageV2 tests can not run in parallel with V1 test.
```

Comment:
Should we be unsetting the environment variable? I think this would prevent us from running V2 tests in parallel with other V2 tests. If we setup the V2 tests to run on their own machine, then we would just need to set it.

Reply:
I see, PFS V1 and V2 tests are running on the same machine. Not running them in parallel is probably fine since they are the faster tests.